python/cudf/cudf/core/_base_index.py

```diff
@@ -88,10 +85,6 @@ def size(self):
         # The size of an index is always its length irrespective of dimension.
         return len(self)
 
-    @property
-    def values(self):
-        return self._values.values
-
```

Comment:
We still want this property to exist, just as an abstract method i.e. no implementation, just raise `NotImplementedError`. Every subclass does need to define a `values` method since (I think) every pandas Index supports `values`.

Reply:
> Every subclass does need to define a values method since (I think) every pandas Index supports values

Correct, although pandas recommends `to_numpy()` for external users